src/WebToPay/CallbackValidator.php

```diff
@@ -53,8 +65,14 @@ public function validateAndParseData(array $requestData) {
             throw new WebToPay_Exception_Callback('"data" parameter not found');
         }
         $data = $requestData['data'];
-
-        $queryString = $this->util->decodeSafeUrlBase64($data);
+        if (isset($requestData['ss1']) || isset($requestData['ss2'])) {
+            $queryString = $this->util->decodeSafeUrlBase64($data);
+        } else {
```

Comment:
Will such request pass condition above `$this->signer->checkSign($requestData)`? I think, not.